decentralized-message-queue/src/DMQ/Configuration/Topology.hs

```diff
@@ -0,0 +1,86 @@
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE NamedFieldPuns    #-}
+{-# LANGUAGE OverloadedStrings #-}
+
```

Comment:
Maybe move this to `ouroboros-network`

Reply:
It would require adding `aeson` as a dependency to `ouroboros-network` which we want to avoid.